cnn_documentation.txt

```diff
@@ -0,0 +1,17 @@
+Documentation of the layer regarding input weight and height to clarify how the input matrix is treated, especially if it is rectangular.
```

Comment:
Adding an extra file to the root directory looks strange to me, do you think you can add the details right after https://github.com/mlpack/mlpack/blob/master/src/mlpack/methods/ann/layer/convolution.hpp#L32

That comes with the main advantage that everything we have there is going to be picked up by doxygen the engine we use to generate the documentation.

Reply:
Just to make sure I am doing the right thing, do you want me to copy the description of the input layer of the CNN in the convolution.hpp file? Is that correct?

Reply:
That is correct.

Reply:
Ok. Done